### DISCUSSION TITLE
Please consider implementing these intresting features..

### DISCUSSION THREAD

**Discussion by Phenom23** [wireapp/wire-webapp#11152] [Ideas]
1. Please add voice messages
2. Add voting functionality in groups 
3. Add emoji section 
4. Add basic reactions per message 
5. Add a quick reply button per message in chats 

( i am talking about the web version of your messenger, i haven't yet used your mobile / desktop app)

Thanks a lot for checking my thoughts, your product is pretty neat, but it would be even greater if you implement these things.

**Comment by jamesonp** [comment]
How about 2FA.  That'd be a nice basic feature to start with.
